.deepsource.toml

```diff
@@ -0,0 +1,14 @@
+version = 1
```

Comment:
We appreciate the config, but we are pretty happy with our current static analysis tooling.  Please omit this file from the PR.